### DISCUSSION TITLE
duck cli: snychronize doesn't clean up orphaned files on S3 bucket

### DISCUSSION THREAD

**Discussion by budachst** [iterate-ch/cyberduck#14985] [Q&A]
Hi,
does anybody use duck cli for synchronizing folders - actually with removing files, that don't longer exist on the source? The modes provided are upload/download/mirror, but there seems to be no way to remove orphaned files.

If that isn't possible with duck, how does one go about this?

**Comment by dkocher** [comment]
This is a known issue [^1].

[^1]: #12171
